package.json

```diff
@@ -14,7 +14,6 @@
   "license": "ISC",
   "dependencies": {
     "axios": "^0.18.0",
-    "dom": "0.0.3",
```

Comment:
If you accidentally install unneeded npm libraries, don't forget to uninstall them later.